packages/lib-classifier/src/plugins/drawingTools/experimental/models/marks/TranscriptionLine/TranscriptionLine.js

```diff
@@ -5,6 +5,7 @@ import { Mark } from '@plugins/drawingTools/models/marks'
 
 const TranscriptionLineModel = types
   .model('TranscriptionLineModel', {
+    toolType: types.literal('transcriptionLine'),
```

Comment:
`toolType` must be added to the drawing code that creates new marks. Otherwise, clicking on the drawing canvas to start a new transcription line will throw an error in MST.

`toolType` should also be added to the tests for drawing a new mark, and also to the drawing tools Readme.

These are the reasons that I would prefer if `toolType` was added to all drawing tools in a separate PR.